packages/sdk/src/client/edition/schedules.ts

```diff
@@ -158,7 +161,7 @@ export async function editionMinterMintIds(
   const { signerOrProvider } = await this.expectSignerOrProvider()
 
   // Query MintConfigCreated event
-  const minterContract = IMinterModule__factory.connect(minterAddress, signerOrProvider)
+  const minterContract = IMinterModuleV2__factory.connect(minterAddress, signerOrProvider)
   const filter = minterContract.filters.MintConfigCreated(editionAddress)
```

Comment:
hmm shouldn't we have this conditional too based on which one user is connecting to?

Reply:
like if a user gives me edition address for contract that was using V1.1 will that work?

Reply:
I believe the create event signature is unchanged since same params, but good call out

I will add a comment here + explicit test

Reply:
@saihaj , addressed here: https://github.com/soundxyz/sdk/pull/215/commits/bb518a486bd74a0a9e0502540eb0412bfe5f1be3